recipes/default.rb

```diff
@@ -62,7 +62,7 @@
 end
 
 influxdb_config node['influxdb']['config_file_path'] do
-  config node['influxdb']['config']
+  config lazy { node['influxdb']['config'] }
```

Comment:
you actually can set the attributes in a recipe during compile phase.  You would only need the lazy operator if you needed to resolve the node attribute during converge phase.

Using an application wrapper cookbook, this is an example of how I enable graphite in the config attribute.
```ruby
node.default['influxdb']['config']['graphite'] = [
  {
    'enabled' => true
  }
]
include_recipe 'influxdb::default'
```